### ISSUE TITLE
Incompatible EOL behavior with other mac clients 

### ISSUE THREAD

**Issue by AdamBD** [ransome1/sleek#111]
**Is it an actual bug?**
I believe so

**Describe the bug**
I am using both sleek as well as a todo.sh CLI program. The problem with using them in parallel due to what seems to be incompatible behavior in EOL characters. After adding entries via Sleek, the first entry added from the CLI will append to the last entry from sleek

Any suggestions on ways to solve this would be greatly appreciated!

**Desktop (please complete the following information):**
TODO.TXT Command Line Interface v2.12.0
MacOS BigSur 11.2.1
zsh 5.8 (x86_64-apple-darwin20.0)
Sleek v -0.9.9

**Comment by ransome1** [comment]
Hi @AdamBD. Could you please give this development version a try and let me know if the issue is still apparent? In this release a new line character is added with each new todo that is created with sleek. I can't say for sure if this works well cross platform, as I havn't tested it properly yet, but please feel free to report back.

https://github.com/ransome1/sleek/releases/tag/v1.0.3

**Comment by ransome1** [comment]
I could not reproduce it and am closing due to lack of response.

**Comment by AdamBD** [comment]
@ransome1 thanks very much for responding and providing me with this solution to check. It DID seem to solve the issue for adding new tasks either via Sleek or the CLI app I am using, however there seems to still be an issue with when items are 'archived' from sleek. After archiving an item any new issue created from the CLI is appended to the previously archived line. Really not sure why that would be happening. 

Do you have any thoughts as to what might be the issue?

**Comment by ransome1** [comment]
Hi @AdamBD . Could you please try out this testing release (https://github.com/ransome1/sleek/releases/tag/v1.0.3-1) and check if the archiving function now works as expected?

**Comment by AdamBD** [comment]
@ransome1 seems to be working great now! Thanks so much for your tireless assistance here. You're awesome!

**Comment by ransome1** [comment]
You'Re very welcome
